test-support/helpers/ember-frost-core.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable ember-standard/destructure */
```

Comment:
The new destructure rule wants all Ember classes destructured except String and Object. In the case of this file it does:
```js
const assign = Object.assign || Ember.assign || Ember.merge
```
in order to support older versions of Ember. Without this rule disabled it would complain that `Ember.assign` and `Ember.merge` are not destructured.